### PR TITLE
Update blockchain/types/accountkey/doc.go

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

Update blockchain/types/accountkey/doc.go

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

The relationship with other packages is on the atlassian page.

### CODE REVIEW

File: blockchain/types/accountkey/doc.go [hunk 1]
```diff
@@ -14,5 +14,38 @@
...
+AccountKey related functions and variables are defined in the files listed below
+  - account_key.go                    : Defines the AccountKey type and the AccountKey interface, and defines the functions related to AccountKey.
+  - account_key_fail.go               : AccountKeyFail is defined according to the AccountKey interface.
+  - account_key_legacy.go             : AccountKeyLegacy is defined according to the AccountKey interface.
+  - account_key_nil.go                : AccountKeyNil is defined according to the AccountKey interface.
+  - account_key_public.go             : AccountKeyPublic is defined according to the AccountKey interface.
+  - account_key_role_based.go         : AccountKeyRoleBased is defined according to the AccountKey interface.
+  - account_key_serializer.go         : AccountKeySerializer is defined for serialization of AccountKey.
+  - account_key_weighted_multi_sig.go : AccountKeyWeightedMultiSig is defined according to the AccountKey interface.
+  - public_key.go                     : PublicKeySerializable is defined to serialize PublicKey.
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - public_key.go                     : PublicKeySerializable is defined for serialization of PublicKey.
```
Just for the consistency of description with line 43

File: blockchain/types/accountkey/doc.go [hunk 2]
```diff
@@ -14,5 +14,38 @@
...
+  - AccountKeyTypeWeightedMultiSig
+  - AccountKeyTypeRoleBased
+
+Each AccountKey type implements the AccountKey interface.
+
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey type and the AccountKey interface, and defines the functions related to AccountKey.
```
Comment (jeonghonah) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key.go                    : Defines the AccountKey type, the AccountKey interface and the functions related to AccountKey.
```

File: blockchain/types/accountkey/doc.go [hunk 3]
```diff
@@ -14,5 +14,38 @@
...
+  - account_key_fail.go               : AccountKeyFail is defined according to the AccountKey interface.
+  - account_key_legacy.go             : AccountKeyLegacy is defined according to the AccountKey interface.
```
Comment (KimKyungup) [inline blockchain/types/accountkey/doc.go]: How about write like this? Just my opinion. :)
`An AccountKey for legacy key is defined.`
`An AccountKey for nil key is defined.`
Also, if legacy key, nil key .. can be explained, it is better.

Comment (jimni1222) [reply blockchain/types/accountkey/doc.go]: Description is edited ! :) Please take a look !

File: blockchain/types/accountkey/doc.go [hunk 4]
```diff
@@ -14,5 +14,38 @@
...
+  - AccountKeyTypeRoleBased
+
+Each AccountKey type implements the AccountKey interface.
+
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey type, the AccountKey interface and the functions related to AccountKey.
+  - account_key_fail.go               : An AccountKey for fail key is defined. If an account has the fail key, the transaction validation process always fails.
```
Comment (KimKyungup) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_fail.go               : An AccountKey for fail key is defined. If an account has the fail key, the account's transaction validation process always fails.
```

File: blockchain/types/accountkey/doc.go [hunk 5]
```diff
@@ -14,5 +14,38 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package accountkey contains types related to keys in Klaytn accounts.
+/*
+The package accountkey implements the AccountKey used in Klaytn.
```
Comment (jeonghonah) [inline blockchain/types/accountkey/doc.go]:
```suggestion
Package accountkey implements the AccountKey used in Klaytn.
```

File: blockchain/types/accountkey/doc.go [hunk 6]
```diff
@@ -14,5 +14,38 @@
...
+  - AccountKeyTypeWeightedMultiSig
+  - AccountKeyTypeRoleBased
+
+Each AccountKey type implements the AccountKey interface.
+
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey type, the AccountKey interface and the functions related to AccountKey.
```
Comment (jeonghonah) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key.go                    : Defines the AccountKey types, the AccountKey interface and the functions related to AccountKey.
```

File: blockchain/types/accountkey/doc.go [hunk 7]
```diff
@@ -14,5 +14,38 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package accountkey contains types related to keys in Klaytn accounts.
+/*
+Package accountkey implements the AccountKey used in Klaytn.
+Inside the package are the types, functions and interfaces associated with the AccountKey.
```
Comment (jeonghonah) [inline blockchain/types/accountkey/doc.go]:
```suggestion
Inside the package, there are the types, functions and interfaces associated with the AccountKey.
```

File: blockchain/types/accountkey/doc.go [hunk 8]
```diff
@@ -14,5 +14,38 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package accountkey contains types related to keys in Klaytn accounts.
+/*
+Package accountkey implements the AccountKey used in Klaytn.
+Inside the package, the types, functions and interfaces associated with the AccountKey are defined.
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
Inside the package, types, functions and interfaces associated with the AccountKey are defined.
```

File: blockchain/types/accountkey/doc.go [hunk 9]
```diff
@@ -14,5 +14,38 @@
...
+The AccountKey types used in Klaytn are as follows:
+  - AccountKeyTypeNil
+  - AccountKeyTypeLegacy
+  - AccountKeyTypePublic
+  - AccountKeyTypeFail
+  - AccountKeyTypeWeightedMultiSig
+  - AccountKeyTypeRoleBased
+
+Each AccountKey type implements the AccountKey interface.
+
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
```
if it is not intentional...

File: blockchain/types/accountkey/doc.go [hunk 10]
```diff
@@ -14,5 +14,38 @@
...
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey types, the AccountKey interface and the functions related to AccountKey.
+  - account_key_fail.go               : An AccountKey for fail key is defined. If an account has the fail key, the account's transaction validation process always fails.
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_fail.go               : An AccountKey for AccountKeyFail type is defined. If an account has the fail key, the account's transaction validation process always fails.
```

File: blockchain/types/accountkey/doc.go [hunk 11]
```diff
@@ -14,5 +14,38 @@
...
+  - account_key_legacy.go             : An AccountKey for legacy key is defined.
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_legacy.go             : An AccountKey for AccountKeyLegacy type is defined.
```

File: blockchain/types/accountkey/doc.go [hunk 12]
```diff
@@ -14,5 +14,38 @@
...
+  - account_key_nil.go                : An AccountKey for nil key is defined.
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_nil.go                : An AccountKey for AccountKeyNil type is defined.
```

File: blockchain/types/accountkey/doc.go [hunk 13]
```diff
@@ -14,5 +14,38 @@
...
+  - account_key_public.go             : An AccountKey for public key is defined.
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_public.go             : An AccountKey for AccountKeyPublic type is defined. If an account contains a public key as an account key, the public key will be used in the account's transaction validation process.
```

File: blockchain/types/accountkey/doc.go [hunk 14]
```diff
@@ -14,5 +14,37 @@
...
+  - account_key.go                    : Defines the AccountKey types, the AccountKey interface and the functions related to AccountKey.
+  - account_key_fail.go               : An AccountKey for AccountKeyFail type is defined. If an account has the fail key, the account's transaction validation process always fails.
+  - account_key_legacy.go             : An AccountKey for AccountKeyLegacy type is defined. If an account has the legacy key, the account's key pair should be coupled with its address.
+  - account_key_nil.go                : An AccountKey for AccountKeyNil type is defined. The nil key is used only for TxTypeAccountUpdate transactions representing an empty key.
+  - account_key_public.go             : An AccountKey for AccountKeyPublic type is defined. If an account contains a public key as an account key, the public key will be used in the account's transaction validation process.
+  - account_key_role_based.go         : An AccountKey for role based key is defined.
+  - account_key_serializer.go         : AccountKeySerializer is defined for serialization of AccountKey.
+  - account_key_weighted_multi_sig.go : An AccountKey for weighted multi sig key is defined.
```
Comment (jimni1222) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_weighted_multi_sig.go : An AccountKey for AccountKeyWeightedMultiSig type is defined.
```

File: blockchain/types/accountkey/doc.go [hunk 15]
```diff
@@ -14,5 +14,37 @@
...
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey types, the AccountKey interface and the functions related to AccountKey.
+  - account_key_fail.go               : An AccountKey for AccountKeyFail type is defined. If an account has the fail key, the account's transaction validation process always fails.
+  - account_key_legacy.go             : An AccountKey for AccountKeyLegacy type is defined. If an account has the legacy key, the account's key pair should be coupled with its address.
+  - account_key_nil.go                : An AccountKey for AccountKeyNil type is defined. The nil key is used only for TxTypeAccountUpdate transactions representing an empty key.
+  - account_key_public.go             : An AccountKey for AccountKeyPublic type is defined. If an account contains a public key as an account key, the public key will be used in the account's transaction validation process.
+  - account_key_role_based.go         : An AccountKey for role based key is defined.
```
Comment (jimni1222) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined.
```

File: blockchain/types/accountkey/doc.go [hunk 16]
```diff
@@ -14,5 +14,37 @@
...
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey types, the AccountKey interface and the functions related to AccountKey.
+  - account_key_fail.go               : An AccountKey for AccountKeyFail type is defined. If an account has the fail key, the account's transaction validation process always fails.
+  - account_key_legacy.go             : An AccountKey for AccountKeyLegacy type is defined. If an account has the legacy key, the account's key pair should be coupled with its address.
+  - account_key_nil.go                : An AccountKey for AccountKeyNil type is defined. The nil key is used only for TxTypeAccountUpdate transactions representing an empty key.
+  - account_key_public.go             : An AccountKey for AccountKeyPublic type is defined. If an account contains a public key as an account key, the public key will be used in the account's transaction validation process.
+  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined.
```
Comment (jimni1222) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined. AccountKeyRoleBased contains keys that have three roles: RoleTransaction, RoleAccountUpdate, and RoleFeePayer.
```

File: blockchain/types/accountkey/doc.go [hunk 17]
```diff
@@ -14,5 +14,37 @@
...
+  - account_key_fail.go               : An AccountKey for AccountKeyFail type is defined. If an account has the fail key, the account's transaction validation process always fails.
+  - account_key_legacy.go             : An AccountKey for AccountKeyLegacy type is defined. If an account has the legacy key, the account's key pair should be coupled with its address.
+  - account_key_nil.go                : An AccountKey for AccountKeyNil type is defined. The nil key is used only for TxTypeAccountUpdate transactions representing an empty key.
+  - account_key_public.go             : An AccountKey for AccountKeyPublic type is defined. If an account contains a public key as an account key, the public key will be used in the account's transaction validation process.
+  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined. AccountKeyRoleBased contains keys that have three roles: RoleTransaction, RoleAccountUpdate, and RoleFeePayer.
+  - account_key_serializer.go         : AccountKeySerializer is defined for serialization of AccountKey.
+  - account_key_weighted_multi_sig.go : An AccountKey for AccountKeyWeightedMultiSig type is defined.
```
Comment (jimni1222) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_weighted_multi_sig.go : An AccountKey for AccountKeyWeightedMultiSig type is defined. AccountKeyWeightedMultiSig contains Threshold and WeightedPublicKeys .
```

File: blockchain/types/accountkey/doc.go [hunk 18]
```diff
@@ -14,5 +14,38 @@
...
+
+Source Files
+
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey types, the AccountKey interface and the functions related to AccountKey.
+  - account_key_fail.go               : An AccountKey for fail key is defined. If an account has the fail key, the account's transaction validation process always fails.
+  - account_key_legacy.go             : An AccountKey for legacy key is defined.
+  - account_key_nil.go                : An AccountKey for nil key is defined.
+  - account_key_public.go             : An AccountKey for public key is defined.
+  - account_key_role_based.go         : An AccountKey for role based key is defined.
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined. If an account has a role-based key that consists of more than one key, the account's transaction validation process will use one key in the role-based key depends on the transaction type. 
```

File: blockchain/types/accountkey/doc.go [hunk 19]
```diff
@@ -14,5 +14,37 @@
...
+  - account_key_legacy.go             : An AccountKey for AccountKeyLegacy type is defined. If an account has the legacy key, the account's key pair should be coupled with its address.
+  - account_key_nil.go                : An AccountKey for AccountKeyNil type is defined. The nil key is used only for TxTypeAccountUpdate transactions representing an empty key.
+  - account_key_public.go             : An AccountKey for AccountKeyPublic type is defined. If an account contains a public key as an account key, the public key will be used in the account's transaction validation process.
+  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined. AccountKeyRoleBased contains keys that have three roles: RoleTransaction, RoleAccountUpdate, and RoleFeePayer.
+  - account_key_serializer.go         : AccountKeySerializer is defined for serialization of AccountKey.
+  - account_key_weighted_multi_sig.go : An AccountKey for AccountKeyWeightedMultiSig type is defined. AccountKeyWeightedMultiSig contains Threshold and WeightedPublicKeys .
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_weighted_multi_sig.go : An AccountKey for AccountKeyWeightedMultiSig type is defined. AccountKeyWeightedMultiSig contains Threshold and WeightedPublicKeys.
```

File: blockchain/types/accountkey/doc.go [hunk 20]
```diff
@@ -14,5 +14,37 @@
...
+  - public_key.go                     : PublicKeySerializable is defined for serialization of public key.
+
+
```
Comment (aidan-kwon) [inline blockchain/types/accountkey/doc.go]:
```suggestion
```

File: blockchain/types/accountkey/doc.go [hunk 21]
```diff
@@ -14,5 +14,36 @@
...
+AccountKey related functions and variables are defined in the files listed below.
+  - account_key.go                    : Defines the AccountKey types, the AccountKey interface and the functions related to AccountKey.
+  - account_key_fail.go               : An AccountKey for AccountKeyFail type is defined. If an account has the fail key, the account's transaction validation process always fails.
+  - account_key_legacy.go             : An AccountKey for AccountKeyLegacy type is defined. If an account has the legacy key, the account's key pair should be coupled with its address.
+  - account_key_nil.go                : An AccountKey for AccountKeyNil type is defined. The nil key is used only for TxTypeAccountUpdate transactions representing an empty key.
+  - account_key_public.go             : An AccountKey for AccountKeyPublic type is defined. If an account contains a public key as an account key, the public key will be used in the account's transaction validation process.
+  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined. AccountKeyRoleBased contains keys that have three roles: RoleTransaction, RoleAccountUpdate, and RoleFeePayer.
```
Comment (jimni1222) [inline blockchain/types/accountkey/doc.go]:
```suggestion
  - account_key_role_based.go         : An AccountKey for AccountKeyRoleBased type is defined. AccountKeyRoleBased contains keys that have three roles: RoleTransaction, RoleAccountUpdate, and RoleFeePayer. If an account has a role-based key that consists of more than one key, the account's transaction validation process will use one key in the role-based key depends on the transaction type. 
```

File: blockchain/types/accountkey/doc.go (hunk 18 above)
Comment (jimni1222) [reply blockchain/types/accountkey/doc.go]: This is applied seperately !!ㅎㅎ

**Comment by jimni1222** [comment]
@jeonghonah @nogang @KimKyungup Can you take a look with this PR again? ;-; I missed approve because of changing...

**Pull request merged**